packages/aws-rfdk/lib/deadline/lib/repository.ts

```diff
@@ -923,6 +979,13 @@ export class Repository extends Construct implements IRepository {
       version.linuxFullVersionString(),
     ];
 
+    if (this.secretsManagementSettings.enabled) {
+      installerArgs.push('true');
+      installerArgs.push(Stack.of(this).region);
```

Comment:
Will this work if we do `Stack.of(this.secretsManagementCredentials).region`? It would protect us if for some reason the Secret the customer provided was in a different region than this Stack.